cpp/include/cudf/scalar/scalar_factories.hpp

```diff
@@ -126,10 +126,11 @@ std::unique_ptr<scalar> make_default_constructed_scalar(data_type type);
 template <typename T>
 std::unique_ptr<scalar> make_fixed_width_scalar(
   T value,
+  bool valid                          = true,
   cudaStream_t stream                 = 0,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource())
 {
-  return std::make_unique<scalar_type_t<T>>(value, true, stream, mr);
+  return std::make_unique<scalar_type_t<T>>(value, valid, stream, mr);
```

Comment:
The reason this function didn't have a `valid` parameter is that it is assumed if you are creating a scalar from a `value` then it is inherently valid.

Reply:
Are you requesting that I revert this back? I liked this vs a branch based on validity when creating the scalar.

Reply:
How can you create a scalar from a value that isn't valid? It doesn't quite make sense to me.

Reply:
The binary op will return a min and a max value along with validity. If you pass an entirely null column to minmax, it will return values that are invalid, but values the same. I was just looking to avoid something like this:
```
std::unique_ptr<scalar> min = make_fixed_width_scalar<T>(result.min_val, stream, mr);
min.set_valid(result.min_valid);
std::unique_ptr<scalar> max = make_fixed_width_scalar<T>(result.max_val, stream, mr);
max.set_valid(result.max_valid);
```
or to branch around validity.

I thought it was cleaner to have it in a single line and be able to call it to set a valid or invalid scalar. What would you suggest here?

Reply:
@jrhemstad I'm not sure where we stand with this one. I no longer need this functionality, but I don't know if it isn't desired.

Reply:
If you don't need it, then let's not add it.